### DISCUSSION TITLE
Linking two clusters in a multicluster setup...

### DISCUSSION THREAD

**Discussion by muranant** [linkerd/linkerd2#8952] [Help]
i have linkerd and multi cluster services running in two clusters kind-west and kind-east. All the linkerd multicluster checks says is OK. 
However when i run the following command 
linkerd --context=kind-east multicluster link --cluster-name kind-east  --verbose | kubectl --context=kind-west apply -f -

i see the error 
Error: could not find service account token secret for linkerd-service-mirror-remote-access-default
Usage:
  linkerd multicluster link [flags]

I looked at the ServiceAccount and it is present in my clusters. Wondering why it cannot find a token secret. I dont see specific steps to set that up on the page:
https://linkerd.io/2.11/tasks/multicluster/#linking-the-clusters

Any help is appreciated. 

Thx
Murali

**Comment by adleong** [comment]
I think this may be instance of https://github.com/linkerd/linkerd2/issues/9033.  Are you using kubernetes 1.24?

**Reply by muranant** [reply]
Yes i am using 1.24 and i think the pointer to the issue you mention might be the problem. 

Thanks
